src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,30 @@
+const { ApiError } = require('../exeptions/auth.error');
+const { jwtServices } = require('../services/jwt.services');
+const { usersServices } = require('../services/users.services');
+
+module.exports = {
+  authMiddleware: async (req, _, next) => {
+    const authorization = 'authorization';
+
+    const token =
+      (req.headers[authorization] &&
+        req.headers[authorization].split(' ')[1]) ||
+      req.cookies.token;
+
+    if (!token) {
+      throw ApiError.unauthorized('Authorization token is missing');
```

Comment:
The `ApiError` constructor arguments are incorrectly ordered. Ensure the correct order is `message, status, errors`.